src/sample-mod.c

```diff
@@ -34,7 +34,7 @@ static int module_initialize(char *conf, char isReload) {
 
     local.sun_family = AF_UNIX;
 
-    strcpy(local.sun_path, SOCK_PATH);
+    strlcpy(local.sun_path, SOCK_PATH, sizeof(local.sun_patch));
```

Comment:
I missed it in a hurry as my train journey way coming to an end. Thanks for pushing a fix.